README.md

```diff
@@ -446,6 +447,34 @@ permission.update() # Update!!
 
 ```
 
+## Token
+https://www.jfrog.com/confluence/display/RTF5X/Access+Tokens#AccessTokens-RESTAPI
+```python
+from requests.auth import HTTPBasicAuth
+from artifactory import ArtifactoryPath
+from dohq_artifactory import Token
+
+session = ArtifactoryPath('https://artifactory_dns/artifactory', auth=('admin', 'admin_password'), auth_type=HTTPBasicAuth, verify=False)
+
+# Read token for readers group
+group_name="readers"
+scope = "api:* member-of-groups:"+group_name
+token = Token(session,  scope=scope)
+token.read()
+
+# Create token for member of the readers
+group_name="readers"
+scope = "api:* member-of-groups:"+group_name
+subject=group_name
+token = Token(session, scope=scope, username=subject, expires_in=31557600, refreshable=True)
+response=token.create()
+
+print("Readonly token:")
+print("Username: "+token.username)
```

Comment:
Code looks unformatted. 

You should rebase this branch onto current `develop` branch and run `pre-commit run --all-files` - it will fix format.